src/components/Editor/ClientCursor.ts

```diff
@@ -29,53 +31,71 @@ class ClientCursor {
     this.removeCursor();
     const cursorCoords = cm.cursorCoords(cursorPos);
     const cursorElement = document.createElement('span');
-    const size = cursorCoords.bottom - cursorCoords.top;
+    this.height = cursorCoords.bottom - cursorCoords.top;
 
     cursorElement.style.position = 'absolute';
     cursorElement.style.borderLeftStyle = 'solid';
-    cursorElement.style.borderLeftWidth = '2px';
+    cursorElement.style.borderLeftWidth = '2.5px';
     cursorElement.style.borderLeftColor = this.color;
-    cursorElement.style.height = `${size}px`;
+    cursorElement.style.height = `${this.height}px`;
     cursorElement.style.padding = '0px';
 
-    const nameElement = document.createElement('span');
-    nameElement.textContent = this.id;
-    nameElement.style.position = 'absolute';
-    nameElement.style.top = `-${size}px`;
-    nameElement.style.left = `-2px`;
-    nameElement.style.backgroundColor = this.color;
-    nameElement.style.padding = '1px 4px';
-    nameElement.style.borderRadius = '4px 4px 4px 0px';
-    nameElement.style.color = invert(this.color, true);
-    nameElement.style.animationDuration = `${duration}s`;
-    nameElement.style.animationDelay = `${delay}s`;
-    nameElement.className = 'text-remove';
-
-    cursorElement.appendChild(nameElement);
-
-    this.removeNameReserve(nameElement);
+    this.showCursorNameReserve(cursorElement);
     this.marker = cm.setBookmark(cursorPos, {
       widget: cursorElement,
       insertLeft: true,
     });
   }
 
-  updateLine(cm: CodeMirror.Editor, fromPos: CodeMirror.Position, toPos: CodeMirror.Position) {
+  updateLine(
+    cm: CodeMirror.Editor,
+    fromPos: CodeMirror.Position,
+    toPos: CodeMirror.Position,
+  ) {
     this.removeLine();
     this.lineMarker = cm.getDoc().markText(fromPos, toPos, {
       css: `background-color : ${this.color}; opacity:0.7`,
     });
   }
 
+  // when user's cursor hover, show name
+  showCursorNameReserve(parentEl: Element) {
```

Comment:
I think the `showCursorNameReserve` method can be a `private` method. Can you confirm?

Reply:
@dc7303 Thank you for your review. I think so! I gonna modify that